src/credentials/sso/ssoCache.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ClientRegistration } from './clientRegistration'
+import { AccessToken } from './accessToken'
+
+export interface SsoCache {
+    loadClientRegistration(ssoRegion: string): ClientRegistration | null
```

Comment:
`null` is banned by eslint rule, used `undefined`

Reply:
nulls have been replaced with undefined